Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -0,0 +1,42 @@
+package org.firstinspires.ftc.team6220_PowerPlay;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.DcMotor;
+
+abstract public class BaseOpMode extends LinearOpMode {
```

Comment:
I think it should be public first, then abstract

Reply:
There's no difference iirc